pkg/yurthub/cachemanager/cache_manager_test.go

```diff
@@ -1047,35 +1046,6 @@ func TestCacheListResponse(t *testing.T) {
 				},
 			},
 		},
-		"list runtimeclasses with no objects": {
-			group:   "node.k8s.io",
-			version: "v1beta1",
-			key:     "kubelet/runtimeclasses",
-			inputObj: runtime.Object(
-				&nodev1beta1.RuntimeClassList{
-					TypeMeta: metav1.TypeMeta{
-						APIVersion: "node.k8s.io/v1beta1",
-						Kind:       "RuntimeClassList",
-					},
-					ListMeta: metav1.ListMeta{
-						ResourceVersion: "12",
-					},
-					Items: []nodev1beta1.RuntimeClass{},
-				},
-			),
-			userAgent:  "kubelet",
-			accept:     "application/json",
-			verb:       "GET",
-			path:       "/apis/node.k8s.io/v1beta1/runtimeclasses",
-			resource:   "runtimeclasses",
-			namespaced: false,
-			expectResult: struct {
-				err  bool
-				data map[string]struct{}
-			}{
-				data: map[string]struct{}{},
-			},
-		},
```

Comment:
It's need to keep these test case.

Reply:
The PR submitted recently recovered this test case.